[aircrack-ng] error: while loading shared libraries: libssl.so.1.0.0

## Description
ISSUE: I'm currently trying to build some cyber-security oriented projects using ChromeOS 
in order to test somethings in my home lab, I need aircrack-ng (along with some other tools listed via 'crew search' ) to be functional however upon 
downloading I recieve the error msg: ./aircrack-ng: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory (see attached image as well) 

If you executed a specific command:
```
./aircrack-ng 
```

Output:
```
./aircrack-ng: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory
```

## Versions and system information
chromebrew: 1.7.5

ChromeOS: 
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={CCBD0510-9FD5-F4B7-53CA-3EBE6CEA52D3}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={CCBD0510-9FD5-F4B7-53CA-3EBE6CEA52D3}
CHROMEOS_RELEASE_BOARD=gnawty-signed-mp-v3keys
CHROMEOS_RELEASE_BRANCH_NUMBER=94
CHROMEOS_RELEASE_BUILDER_PATH=gnawty-release/R88-13597.94.0
CHROMEOS_RELEASE_BUILD_NUMBER=13597
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=88
CHROMEOS_RELEASE_DESCRIPTION=13597.94.0 (Official Build) stable-channel gnawty 
CHROMEOS_RELEASE_KEYSET=mp-v3
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=13597.94.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=13597.94.0

Model: YOUR MODEL HERE

Architecture: `uname -m`

## Addtional information
Mention things we might need to know. Like:

What I think needs to be done:
From my research and digging on other forums my understading is that there are two possible solutions. Either (A.) Download the missing module ( libssl.so.1.0.0 in this case)--which I haven't been able to find or do. (B.) Create a symbolic link using 'ln -s' point to the desired/current available module (libssl.so.1.1) 

- [step 1] 'ln -s libssl.so.1.0.0 libssl.so.1.1'
- [ step 2] 'ln -s libcrypto.so.1.0.0 libssl.so.1.1'

However, despite doing this i'm still getting the same error as I did to start.

PS (NOTE): 
Though this is occuring specifically with 'aircrack-ng' tool the issue of 'error: while loading shared libraries: [some-library]', is occuring when attempting to make use of other tools such as 'tcpdump'
![Screenshot 2021-02-21 at 5 18 31 PM](https://user-images.githubusercontent.com/79465201/108731605-54e2f600-74f2-11eb-9eb6-50479f4d433d.png)

---

## That's it
Thank you for submitting your issue.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.

--- a/packages/aircrack_ng.rb
+++ b/packages/aircrack_ng.rb
@@ -3,51 +3,53 @@ require 'package'
 class Aircrack_ng < Package
   description 'Key cracker for the 802.11 WEP and WPA-PSK protocols.'
   homepage 'https://www.aircrack-ng.org'
-  version '1.2-rc4-3'
+  version '1.6-17f9'
   compatibility 'all'
-  source_url 'http://download.aircrack-ng.org/aircrack-ng-1.2-rc4.tar.gz'
-  source_sha256 'd93ac16aade5b4d37ab8cdf6ce4b855835096ccf83deb65ffdeff6d666eaff36'
+  source_url 'https://github.com/aircrack-ng/aircrack-ng/archive/17f971cebdaeea0c86e338bde09a52787fcd6d9c.zip'
+  source_sha256 'd6dc9078f2ab56671f8fb707aa5ddbbec36d1a732cc7ad9a3d2803a5dbbaae5b'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/aircrack_ng-1.2-rc4-3-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/aircrack_ng-1.2-rc4-3-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/aircrack_ng-1.2-rc4-3-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/aircrack_ng-1.2-rc4-3-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/aircrack_ng-1.6-17f9-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/aircrack_ng-1.6-17f9-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/aircrack_ng-1.6-17f9-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/aircrack_ng-1.6-17f9-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: '6597d9d24115513c16113639a86f62fee44f6464d52ce9f18e782c9ca3ff97dd',
-     armv7l: '6597d9d24115513c16113639a86f62fee44f6464d52ce9f18e782c9ca3ff97dd',
-       i686: '52a3a53e68d690426018758cb0b5dbbe692b7ac70d0b42d0b198adb7010dbe42',
-     x86_64: '8331f17911c1eac404a91eefd1a2673b4a2229f3c555155ca7622d534f59ca19',
+  binary_sha256({
+    aarch64: '7c7b369f40e783ebad0786124f109b50ef9db65ae5b5fa725af55712c8eefea4',
+     armv7l: '7c7b369f40e783ebad0786124f109b50ef9db65ae5b5fa725af55712c8eefea4',
+       i686: '27d67a5bbadb60975451f79e5a019a4567f7241519e248ea54050be4e4c554dc',
+     x86_64: '5c7cfe8e71c1dc3f5634027855ebee88d458a08f44bca3ed0a01388f0524524c'
   })
 
-  depends_on 'bison' => :build
-  depends_on 'flex' => :build
   depends_on 'libnl3'
   depends_on 'libpcap'
-  depends_on 'sqlite'
   depends_on 'rfkill'
-  depends_on 'zlibpkg'
 
   def self.build
-    # Need to specify TMPDIR to run automatic configuration tool correctly
-    system "TMPDIR=#{CREW_PREFIX}/tmp make sqlite=true experimental=true"
+    system 'NOCONFIGURE=1 ./autogen.sh'
+    # system "./configure --help"
+    # Could also add more specific optimization options:
+    # --with-static-simd[=x86-sse2|x86-avx|x86-avx2|x86-avx512|ppc-altivec|ppc-power8|arm-neon|arm-asimd],
+    # use specific SIMD implementation at static link, [default=none]
+    # --with-avx512[=yes|no]  use AVX-512F instruction set, [default=no]
+    system "env CFLAGS='-flto=auto' CXXFLAGS='-flto=auto' \
+      LDFLAGS='-flto=auto' \
+      TMPDIR=#{CREW_PREFIX}/tmp \
+      ./configure \
+      #{CREW_OPTIONS} \
+      --program-prefix='' --program-suffix='' \
+      --with-lto \
+      --with-experimental=yes \
+      --with-openssl=#{CREW_PREFIX} \
+      sbindir=#{CREW_PREFIX}/sbin \
+      mandir=#{CREW_PREFIX}/share/man/man1 \
+      smandir=#{CREW_PREFIX}/share/man/man8"
+    system 'make'
   end
 
   def self.install
-    system "make", "DESTDIR=#{CREW_DEST_DIR}",
-      "sqlite=true",
-      "experimental=true",
-      "prefix=#{CREW_PREFIX}",
-      "libdir=#{CREW_LIB_PREFIX}",
-      "bindir=#{CREW_PREFIX}/bin",
-      "sbindir=#{CREW_PREFIX}/sbin",
-      "mandir=#{CREW_PREFIX}/share/man/man1",
-      "smandir=#{CREW_PREFIX}/share/man/man8",
-      "install"
-  end
-
-  def self.postinstall
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}",
+           'install'
     # Fix for [*] Run 'airodump-ng-oui-update' as root (or with sudo) to install or update Airodump-ng OUI file (Internet connection required).
     #
     # Before fix:
@@ -61,15 +63,15 @@ class Aircrack_ng < Package
     # [*] Downloading IEEE OUI file...
     # [*] Parsing OUI file...
     # [*] Airodump-ng OUI file successfully updated
-    system "sed -i 's,/usr/local,,' #{CREW_PREFIX}/sbin/airodump-ng-oui-update"
-    system "sed -i 's,/etc,#{CREW_PREFIX}/etc,g' #{CREW_PREFIX}/sbin/airodump-ng-oui-update"
-    system "sed -i 's,/usr/share,#{CREW_PREFIX}/share,' #{CREW_PREFIX}/sbin/airodump-ng-oui-update"
+    system "sed -i 's,/usr/local,,' #{CREW_DEST_PREFIX}/sbin/airodump-ng-oui-update"
+    system "sed -i 's,/etc,#{CREW_PREFIX}/etc,g' #{CREW_DEST_PREFIX}/sbin/airodump-ng-oui-update"
+    system "sed -i 's,/usr/share,#{CREW_PREFIX}/share,' #{CREW_DEST_PREFIX}/sbin/airodump-ng-oui-update"
   end
 
   def self.check
-    system "make",
-      "sqlite=true",
-      "experimental=true",
-      "check"
+    system 'make',
+           'sqlite=true',
+           'experimental=true',
+           'check'
   end
 end

